src/views/pages/Lookup.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="Lookup">
+    <h1>{{ title }}</h1>
+    <md-field md-inline>
+      <label>Placeholder</label>
+      <md-input v-model="initial"></md-input>
+    </md-field>
+    <h2>Results</h2>
+    <agritable />
```

Comment:
I would personally probably prototype the page using something like Figma first to get a generic feel for what is usable. [This](http://www.designprinciplesftw.com/collections/10-usability-heuristics-for-user-interface-design) is a good resource that we used from HCI that actually helps a lot

PS: It might look cool to put the table into a card, there is an API on the md-table for exactly that